src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseImportBasicService.java

```diff
@@ -116,6 +116,7 @@ public ProgrammingExercise importProgrammingExerciseBasis(final ProgrammingExerc
         final Map<Long, Long> newTestCaseIdByOldId = importTestCases(templateExercise, importedExercise);
         final Map<Long, Long> newTaskIdByOldId = importTasks(templateExercise, importedExercise, newTestCaseIdByOldId);
         updateTaskExerciseHintReferences(templateExercise, importedExercise, newTaskIdByOldId, newHintIdByOldId);
+        importSubmissionPolicy(importedExercise);
```

Comment:
Why is the order relevant here?
Maybe leave a comment in the code explaining it